app/src/test/java/io/apicurio/registry/events/KafkaEventsTest.java

```diff
@@ -57,8 +57,8 @@
  */
 @QuarkusTest
 @TestProfile(KafkaEventsProfile.class)
-@DisabledIfSystemProperty(named = "kafka.storage", matches = "true")
 @Tag(ApicurioTestTags.DOCKER)
+@Disabled
```

Comment:
I guess this needs some attention?

Reply:
It does but not in this PR. This is something that wasn't working before already. Same with HttpEventsTests, it's not working and it should be addressed, but I don't think it should be done here.